### ISSUE TITLE
[Bug][a11y]: In Windows High Contrast Mode, the tablist is missing its border.

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3255]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Tabs

### Expected behavior

When Windows High Contrast Mode is enabled, the tablist should have a bottom border when tabs are in a horizontal orientation, and a left border when tabs are in a vertical orientation, just like how it behaves in non-HCM settings.

![image](https://github.com/adobe/spectrum-web-components/assets/16927765/286c31c2-e254-4937-8cd0-f0d3e79f7e8b)

![image](https://github.com/adobe/spectrum-web-components/assets/16927765/8af6c1c0-efe0-48bf-a755-49b3c883a8b0)

### Actual behavior

When using Windows High Contrast Mode, the tablist does not display any border, making it challenging to distinguish what component it is.

![image](https://github.com/adobe/spectrum-web-components/assets/16927765/4027d863-948d-4622-a336-4b383d1aafe4)

![image](https://github.com/adobe/spectrum-web-components/assets/16927765/7cc50352-c280-434d-967b-a82bb87938c2)

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/tabs/
2. Open the Chrome browser and navigate to the website you want to test.
3. Open the Chrome DevTools by pressing Ctrl+Shift+I (Windows) or Cmd+Option+I (Mac).
4. Click on the "Rendering" tab.
5. In the Rendering tab, scroll down and find the "Emulate CSS media feature forced-colors" option.
6. Select the "active" option from the drop-down menu.

**Comment by pfulton** [comment]
This is great timing for us: we're currently migrating the Tabs component, and we'll be sure that this gets taken care of during that work. (cc: @mlogsdon18)

**Comment by Westbrook** [comment]
fixed by https://github.com/adobe/spectrum-web-components/pull/3333
